src/cdk/scrolling/virtual-scroll-viewport.scss

```diff
@@ -69,11 +73,7 @@ cdk-virtual-scroll-viewport {
 // set if it were rendered all at once. This ensures that the scrollable content region is the
 // correct size.
 .cdk-virtual-scroll-spacer {
-  position: absolute;
```

Comment:
@spike-rabbit I started looking into why som tests were failing internally with this change, and it looked like some people were setting `display: flex` on the viewport. Without this `position: absolute` the spacer is collapsing and throwing off the scrollbar. 

I think we could definitely question _why_ they're making it `display: flex`, as I don't see any real reason to do that, but I think we could help prevent unexpected breakages by just slapping a `flex: 0 0 auto` on here.

I also notice that below you still have `right: 0` and `left: auto` which can be removed, now that its no longer `position: absolute`